test/source/mock/fes/fes-endpoints.ts

```diff
@@ -43,15 +43,13 @@ export const mockFesEndpoints: HandlersDefinition = {
     }
     throw new HttpClientErr('Not Found', 404);
   },
-  '/api/v1/account/': async ({ }, req) => {
+  '/api/v1/client-configuration': async ({ }, req) => {
+    // ?domain=fes.localhost:8001
+    // ?domain=fes.standardsubdomainfes.test:8001
+    // ?domain=fes.google.mock.flowcryptlocal.test:8001
```

Comment:
Need to see if it will work this way in mock tests. Also, should do an additional check on the domain in this mock endpoint.